### PR TITLE
LPS-70096 color picker ddm field template

### PR DESCRIPTION
**Pull request by pavel-savinov**

### CODE REVIEW

**Comment by pavel-savinov** [comment]
CI failure has nothing to do with the PR.

File: dynamic-data-mapping-service/src/main/resources/META-INF/spring/dynamic-data-mapping-spring.xml [hunk 1]
```diff
@@ -13,6 +13,7 @@
 		<property name="defaultDDMFormFieldValueRenderers">
 			<list>
 				<bean class="com.liferay.dynamic.data.mapping.internal.render.CheckboxDDMFormFieldValueRenderer" />
+				<bean class="com.liferay.dynamic.data.mapping.internal.render.ColorDDMFromFieldValueRenderer" />
```
Comment (rafaprax) [inline dynamic-data-mapping-service/src/main/resources/META-INF/spring/dynamic-data-mapping-spring.xml]: @pavel-savinov Fix the mapping please.

File: dynamic-data-mapping-service/src/main/resources/com/liferay/dynamic/data/mapping/dependencies/ddm/color.ftl [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+		<@liferay_aui.input name=namespacedFieldName type="hidden" value=fieldRawValue>
+			<#if required>
+				<@liferay_aui.validator name="required" />
+			</#if>
+		</@>
+	</div>
+</@>
+
+<@liferay_aui.script use="aui-base,aui-color-picker-popover">
```
Comment (rafaprax) [inline dynamic-data-mapping-service/src/main/resources/com/liferay/dynamic/data/mapping/dependencies/ddm/color.ftl]: @pavel-savinov Could you please move the javascript code for the custom_field.js?

**Comment by rafaprax** [comment]
@pavel-savinov Please don't use squash. Only add commits on top of this branch. Thx

File: dynamic-data-mapping-service/src/main/resources/com/liferay/dynamic/data/mapping/dependencies/ddm/color.ftl (hunk 2 above)
Comment (rafaprax) [inline dynamic-data-mapping-service/src/main/resources/com/liferay/dynamic/data/mapping/dependencies/ddm/color.ftl]: @pavel-savinov Could you please move the javascript code for the ddm_form.js? Thx

**Pull request closed without merging**